allennlp/modules/vision/region_detector.py

```diff
@@ -74,6 +76,19 @@ class RandomRegionDetector(RegionDetector):
     the proposal are a random 10-dimensional vector, and the coordinates are the size of the image.
     """
 
+    def __init__(self, seed: Optional[int] = None):
+        super().__init__()
+        self.random = random.Random(seed)
+
+    def _seeded_random_tensor(self, *shape: int, device) -> torch.FloatTensor:
```

Comment:
Cool! This _may_ be helpful for the non-deterministic dropout layers that were causing (currently commented) test failures in the transformer toolkit. I'll try it out.

Reply:
Cool indeed if this fixes an unrelated problem! Totally uncool from pytorch that we have to do it this way though...